examples/ahiqar-arabic-karshuni-local.html

```diff
@@ -31,6 +31,7 @@
       "item": "Sheet",
       "manifest": "Manuscript"
     },
+    "lang": "en",
```

Comment:
Is this necessary to add? The default behaviour is that it switches automatically to english.

Reply:
no, it's not, true